runelite-client/src/main/java/net/runelite/client/plugins/timers/TimersPlugin.java

```diff
@@ -542,6 +543,10 @@ else if (DEADMAN_HALF_TELEBLOCK_PATTERN.matcher(event.getMessage()).find())
 			{
 				createGameTimer(DMM_HALFTB);
 			}
+			else if (event.getMessage().startsWith(KILLED_TELEBLOCK_OPPONENT_TEXT))
```

Comment:
why `startsWith`? what is the full message?

Reply:
The full message is `"<col=4f006f>Your Tele Block has been removed because you killed (.+).</col>"`, where `(.+)` is the name of the player who teleblocked you.